packages/did-provider-oyd/CHANGELOG.md

```diff
@@ -0,0 +1,207 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# [5.5.0](https://github.com/uport-project/veramo/compare/v5.4.1...v5.5.0) (2023-09-19)
```

Comment:
this line and below should be deleted since it's a new package

Reply:
updated changelog